spec/spec_helper.rb

```diff
@@ -91,6 +91,10 @@
   # with the test execution.
   config.around do |example|
     example.run.tap do
+      # Background thread leak checking is currently too verbose for CI,
+      # slowing down builds to the point of causing CircleCI timeouts.
```

Comment:
JRuby builds are timing out: https://app.circleci.com/pipelines/github/DataDog/dd-trace-rb/3161/workflows/2c837108-5272-42dc-841a-694d76e30b60/jobs/126109
And build console output is over 400MiB 😢

Reply:
400mb of feedback is clearly not helpful. Rather than skipping it entirely in CI, could we count how many errors we've emitted, and after some number, we just stop reporting them, and just include a line at the end of the report with how many errors?

Famously, [valgrind](https://valgrind.org/) does the same:

```
More than 10000000 total errors detected.  I'm not reporting any more.
Final error counts will be inaccurate.  Go fix your program!
```